codegen/transformers.go

```diff
@@ -54,11 +54,24 @@ func defaultGoTypeToSchemaType(v reflect.Type) (schema.ValueType, error) {
 		}
 		return schema.TypeJSON, nil
 	case reflect.Slice:
+		switch v.Elem().Kind() {
+		case reflect.Uint8:
```

Comment:
Shouldn't this be in `defaultGoTypeToSchemaType`?

Reply:
`defaultGoTypeToSchemaType` will return `TypeInt` in this case, effectively resulting in `TypeIntArray` returned

Reply:
there's no `schema.TypeByte`, so no.